core/src/test/java/org/apache/accumulo/core/clientImpl/TableOperationsHelperTest.java

```diff
@@ -267,7 +269,7 @@ void check(TableOperationsHelper t, String tablename, String[] values) throws Ex
       String[] parts = value.split("=", 2);
       expected.put(parts[0], parts[1]);
     }
-    Map<String,String> actual = Map.copyOf(t.getConfiguration(tablename));
```

Comment:
This one doesn't matter. Map equality doesn't depend on order, only equivalent mappings. It'd be better to avoid the use of Guava's method here.